.github/workflows/release-please.yaml

```diff
@@ -21,11 +21,12 @@ jobs:
           draft-pull-request: true
           release-type: simple
           changelog-types: '[
-                       {"type": "feat", "section": "Enhancements", "hidden": false },
-                       {"type": "fix", "section": "Bug Fixes", "hidden": false },
-                       {"type": "build", "section": "Maintenance", "hidden": false },
-                       {"type": "chore", "section": "Maintenance", "hidden": false },
-                       {"type": "test", "section": "Maintenance", "hidden": false },
-                       {"type": "refactor", "section": "Maintenance", "hidden": false },
-                       {"type": "docs", "section": "Documentation", "hidden": false }
-                     ]'
+            {"type": "build", "section": "Maintenances",
+            {"type": "chore", "section": "Maintenances",
+            {"type": "docs", "section": "Documentation",
+            {"type": "feat", "section": "Enhancements",
+            {"type": "fix", "section": "Bug Fixes",
+            {"type": "perf", "section": "Performance Improvements",
+            {"type": "refactor", "section": "Maintenances",
+            {"type": "test", "section": "Tests",
```

Comment:
closing brackets missing?

Reply:
Too much mechanic changes and on the 3rd one I failed. Thanks for spotting this!